src/nearby-stop-places/NearbyStopPlacesScreenComponent.tsx

```diff
@@ -180,14 +185,37 @@ export const NearbyStopPlacesScreenComponent = ({
       )}
     >
       <ScreenReaderAnnouncement message={loadAnnouncement} />
-      <StopPlaces
-        header={getListDescription()}
-        stopPlaces={orderedStopPlaces}
-        navigateToPlace={onSelectStopPlace}
-        testID="nearbyStopsContainerView"
-        location={location}
-        isLoading={isLoading}
-      />
+      {hasLocationPermission ? (
```

Comment:
```suggestion
      {hasLocationPermission || !!location ? (
```
Also check if a location has been selected from favorites

Reply:
Flipping the condition could also be an option:
`{!hasLocationPermission ? <EmptyState ... /> : <StopPlaces ... />`